src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/instantiate/instantiation/sagas/instantiate/index.ts

```diff
@@ -21,7 +21,18 @@ function* instantiationSaga(): SagaIterator {
       const selectedParty: IParty = yield select(SelectedPartySelector);
 
       // Creates a new instance
-      const instanceResponse: any = yield call(post, getCreateInstancesUrl(selectedParty.partyId));
+      let instanceResponse: any;
```

Comment:
Could we define an interface for this response?

Reply:
Looks like we actually have one, although it looks slightly outdated. I will update the interface with missing fields and use that here